packages/console/src/ds-components/Select/index.tsx

```diff
@@ -26,7 +26,7 @@ type Props<T> = {
   readonly options: Array<Option<T>>;
   readonly onChange?: (value?: T) => void;
   readonly isReadOnly?: boolean;
-  readonly error?: string | boolean;
+  readonly error?: string | boolean | ReactElement;
```

Comment:
Use `ReactNode` instead? Since `ReaceNode` includes `string` and `boolean`

Reply:
Or maybe if you want to be specific, like what you did later in Line 153, you can only allow `string | ReactElement` maybe?